CMakeLists.txt

```diff
@@ -186,8 +188,10 @@ endif ()
 include ( CMakePrintHelpers ) # for cmake_print_properties() and cmake_print_variables()
 include ( TestInline )
 include ( TestVLA )
+if ( NOT HAIKU )
 include ( TestBigEndian )
 test_big_endian ( WORDS_BIGENDIAN )
+endif ( NOT HAIKU )
```

Comment:
Why is that if-ed out?

Reply:
The macros were broken on Haiku I believe.